helm/examples/nfs/values-nfs.yaml

```diff
@@ -5,16 +5,57 @@
 # PVC's required by HPCC.
 # The pvc names are dynamic based on the install name.
 # In this example the install name was "nfsstorage"
-
 storage:
+  planes:
+  - name: dali
+    pvc: dali-nfsstorage-hpcc-nfs-pvc
+    prefix: "/var/lib/HPCCSystems/dalistorage"
+    labels:
+    - dali
+
+  - name: dll
+    pvc: dll-nfsstorage-hpcc-nfs-pvc
+    prefix: "/var/lib/HPCCSystems/queries"
+    labels:
+    - dll
+
+  - name: sasha
+    pvc: sasha-nfsstorage-hpcc-nfs-pvc
+    prefix: "/var/lib/HPCCSystems/sasha"
+    labels:
+    - sasha
+
+  - name: data
+    pvc: data-nfsstorage-hpcc-nfs-pvc
+    prefix: "/var/lib/HPCCSystems/hpcc-data"
+    labels:
+    - data # NB: if not set, this would be the default
+
+  - name: dropzone
```

Comment:
Adding dropzones to [all] examples should be treated as a separate issue.

Reply:
Agree